runtime/jcl/common/jclcinit.c

```diff
@@ -634,6 +634,10 @@ initializeRequiredClasses(J9VMThread *vmThread, char* dllName)
 	if (0 != vmFuncs->addHiddenInstanceField(vm, "java/lang/invoke/MutableCallSite", "invalidationCookie", "J", &vm->mutableCallSiteInvalidationCookieOffset)) {
 		return 1;
 	}
+
+	if (0 != vmFuncs->addHiddenInstanceField(vm, "java/lang/invoke/MethodHandle", "jitVMEntryKeepAlive", "Ljava/lang/invoke/MemberName;", &vm->jitVMEntryKeepAliveOffset)) {
+		return 1;
+	}
```

Comment:
Minor formatting nitpick: indentation for lines 634-640 is different than the surrounding code.

Reply:
The indentation is actually exactly the same, which you should be able to see in an editor (e.g. with vim's `set list listchars=tab:>-`) by checking out this commit, copy-pasting from this diff, or downloading the raw contents of [jclcinit.c in this commit](https://github.com/jdmpapin/openj9/blob/4e9032ce962013c3c4598b002554a47d9a1b386e/runtime/jcl/common/jclcinit.c). Here's a [snippet that shows the indentation lining up visually](https://github.com/jdmpapin/openj9/blob/4e9032ce962013c3c4598b002554a47d9a1b386e/runtime/jcl/common/jclcinit.c#L625-L641) at least, though it doesn't visualize the whitespace

The apparent formatting problem looks to be some rendering quirk at the intersection of GitHub diffs, tabs, and Firefox

Reply:
Agreed, it's a browser issue. In Chrome, I do not notice the indentation issue.